### DISCUSSION TITLE
Problem after upload firmware

### DISCUSSION THREAD

**Discussion by dejzar** [NicoHood/HID#274] [Q&A]
Hi, I'm upload this sketch https://prnt.sc/1192zzl to my arduino pro micro and after reconnect my board don't defined https://prnt.sc/119351q (Unknown USB device (device descriptor request failed)). Please help me. I'm try to install driver from program files/arduino/drivers.

**Comment by Ekristoffe** [comment]
I know it's a long shot but have you selected the right speed in the arduino IDE ?
Under Tool / Processor
